docker-hub/publish/insights-analytics.md

```diff
@@ -4,11 +4,11 @@ description: Provides usage statistics of your images on Docker Hub.
 keywords: docker hub, hub, insights, analytics, api, verified publisher
 ---
 
-Insights and analytics provides usage analytics for your organization's images
-on Docker Hub. With this tool, you have self-serve access to metrics as both raw
-data and summary data for a desired time span. You can view how many times your
-images have been pulled by tag or by digest, and get breakdowns by geolocation,
-cloud provider, and client (user agent).
+Insights and analytics provides usage analytics for your Docker Verified
+Publisher (DVP) images on Docker Hub. With this tool, you have self-serve access
+to metrics as both raw data and summary data for a desired time span. You can
+view number of image pulls by tag or by digest, and get breakdowns by
+geolocation, cloud provider, and client, and more.
```

Comment:
```suggestion
geolocation, cloud provider, client, and more.
```

Not sure if the double `and` was intended, pointing it out to be sure

Reply:
it was not intended, thanks 👀